src/braket/experimental/autoqasm/README.md

```diff
@@ -136,10 +142,100 @@ Please tag your question with "Amazon Braket" and mention AutoQASM in the questi
+quantum programs: circuits, gates and pulses. Higher-level algorithm
+libraries could be implemented using AutoQASM and benefit from modular
+AutoQASM functionality such as subroutines.
+
+### 2. What is the relationship between AutoQASM and OpenQASM?
+
+AutoQASM can be seen as implementing a builder pattern for OpenQASM. It
+allows you serialize your program to OpenQASM with `Program.to_ir()`. The
+interface is not strongly tied to OpenQASM so we could serialize to other
+formats in the future.
+
+In other words, AutoQASM is a quantum programming interface built in Python.
+OpenQASM is a quantum assembly language, often used as a serialization format
+for quantum programming frameworks and quantum hardware providers. We can
+represent a quantum program equivalently in either format, but using AutoQASM
+allows one to also make use of Python, including the Amazon Braket SDK.
+
+### 3. What is the relationship between AutoQASM and the Amazon Braket SDK?
+
+AutoQASM lives alongside the Amazon Braket SDK as an experimental feature
+branch. It supplements the program building experience and integrates with
+Amazon Braket SDK features. For instance, one can build a program through
+AutoQASM, using Amazon Braket SDK device objects, and then run the program
+with the SDK.
+
+Quantum programs are serialized to OpenQASM before executing on Amazon
+Braket, and AutoQASM programs can be serialized to OpenQASM. Thus, we have a
+very lightweight integration to run AutoQASM programs through the Amazon
+Braket SDK.
+
+### 4. Does AutoQASM support other providers beyond Amazon Braket?
+
+Yes. AutoQASM serializes to OpenQASM, and so it is applicable to any library
+or QPU that supports OpenQASM. We do have features that use the Amazon Braket
+SDK, such as [device-specific validation](../../../../examples/autoqasm/4_Native_programming.ipynb).
+Because AutoQASM is open-source, anyone could
+build similar integrations for another service. Reach out if you're
+interested in doing this and would like support.
+
+
+### 5. Does AutoQASM offer special support for device-specific programming?
+
+Yes, AutoQASM has device-specific validation to support native programming.
+We plan to expand this functionality in the future. Learn more with our
+[native programming example notebook](../../../../examples/autoqasm/4_Native_programming.ipynb).
+
+### 6. Do the devices available through Amazon Braket support all of AutoQASM's features?
+
+No, for example, the `reset` instruction is not supported by all devices. In
+general, different QPUs and QHPs support different sets of features, so
+AutoQASM will often support features that a particular device doesn't
+support. We intend that AutoQASM will eventually be able to generate any
+program representable by OpenQASM 3.0, with additional Python-side features
+such as validation and visualization.
+
+### 7. Is there a difference between classical conditionals and quantum conditionals?
```

Comment:
Is this more confusing than it's worth?

Reply:
I'm not sure it's really a question about AutoQASM. Your answer does start to explain the difference between trace-time and runtime conditionals, but that's also not what the question is asking. Maybe something like this could go in an example notebook that introduces control flow and mid-circuit measurement?